src/run.ts

```diff
@@ -0,0 +1,94 @@
+import { Argv } from 'yargs';
+import { Command, Helper, OptionsHelper } from '@dojo/interfaces/cli';
+import { join, relative, resolve } from 'path';
+import * as chalk from 'chalk';
+import createDir from '@dojo/cli-create-app/createDir';
+import renderFiles from '@dojo/cli-create-app/renderFiles';
+
+import register from './register';
+import run from './run';
+
+const pkgDir: any = require('pkg-dir');
+const packagePath = pkgDir.sync(__dirname);
+
+/**
+ * @type CreateWidgetArgs
+ *
+ * Arguments that can be passed when creating a widget using the Dojo CLI
+ *
+ * @property component       Generate a Custom Element wrapper
+ * @property name            Name used for the generated widget
+ * @property styles          Path to place generated CSS files
+ * @property tests           Path to place generated test files
+ */
+export interface CreateWidgetArgs extends Argv {
+	component: boolean;
+	name: string;
+	styles: string;
+	tests: string;
+}
+
+function getDirectoryNames(args: CreateWidgetArgs, folderName: string) {
+	const dirs = [ folderName ];
+
+	if (!args.styles) {
+		dirs.push(`${folderName}/styles`);
+	}
+
+	if (!args.tests) {
```

Comment:
The `if` or `else` is not covered by the unit tests

Reply:
I'm showing 100% coverage all around.